convention.md

```diff
@@ -3,10 +3,35 @@
 Version: **<!--VERSION-->x.x.x<!--VERSION-->**
 Date: **<!--DATE-->01. Jan 2000<!--DATE-->**
 
+## Introduction
+
+​The Homie convention is an open standard framework designed to facilitate the
+communication and integration of Internet of Things (IoT) devices using the
+[MQTT protocol](https://mqtt.org).
+
+In particular, the Homie convention defines a consistent topic structure and
+messaging format that enables devices to represent themselves, their data and
+their supported controls/commands in a uniform manner, enabling automatic
+discovery.
+
+## Definitions
+
+- **Device**: a physical appliance or logical entity, such as a car, a coffee
+  machine or a protocol bridge, whose representation is published to an MQTT
+  broker.
+- **Controller**: a software solution that discovers and interacts with
+  *devices* through one or more MQTT brokers.
+
+Note that, for brevity and simplicity, the word &quot;device&quot; is often used
+to denote the combination of an appliance, the computer managing the appliance's
+MQTT representation and the representation itself.
+
 ## MQTT Restrictions
 
 Homie communicates through [MQTT](http://mqtt.org) and is hence based on the basic principles of MQTT topic publication and subscription.
 
+Note that there can be more than one Controller interacting with Devices afferent to the same broker.
+
```

Comment:
Do we need this? this is implicit in the fact that MQTT is the underlying protocol used.

Reply:
Done in https://github.com/homieiot/convention/pull/325/commits/01c93f96ef5ba37f5e42af47e57c45f4ec2ea3ef .